client/a8c-for-agencies/sections/sites/sites-dashboard/index.tsx

```diff
@@ -0,0 +1,221 @@
+} from 'calypso/jetpack-cloud/sections/agency-dashboard/sites-overview/sites-dataviews/interfaces';
+import { AgencyDashboardFilterMap } from 'calypso/jetpack-cloud/sections/agency-dashboard/sites-overview/types';
+import { useSelector } from 'calypso/state';
+import { checkIfJetpackSiteGotDisconnected } from 'calypso/state/jetpack-agency-dashboard/selectors';
+import useProductsQuery from 'calypso/state/partner-portal/licenses/hooks/use-products-query';
+import { getIsPartnerOAuthTokenLoaded } from 'calypso/state/partner-portal/partner/selectors';
+import { A4A_SITES_DASHBOARD_DEFAULT_CATEGORY } from '../constants';
+import SitesDashboardContext from '../sites-dashboard-context';
+
+import './style.scss';
+
+export default function SitesDashboard() {
+	const jetpackSiteDisconnected = useSelector( checkIfJetpackSiteGotDisconnected );
+
+	//const { hideListing } = useContext( SitesDashboardContext );
+	const { selectedCategory: category, setSelectedCategory: setCategory } =
+		useContext( SitesDashboardContext );
+
+	const { selectedSiteUrl } = useContext( SitesDashboardContext );
+	const { selectedSiteFeature } = useContext( SitesDashboardContext );
+
+	const filtersMap = useMemo< AgencyDashboardFilterMap[] >(
+		() => [
+			{ filterType: 'all_issues', ref: 1 },
+			{ filterType: 'backup_failed', ref: 2 },
+			{ filterType: 'backup_warning', ref: 3 },
+			{ filterType: 'threats_found', ref: 4 },
+			{ filterType: 'site_disconnected', ref: 5 },
+			{ filterType: 'site_down', ref: 6 },
+			{ filterType: 'plugin_updates', ref: 7 },
+		],
+		[]
+	);
+
+	const isLargeScreen = isWithinBreakpoint( '>960px' );
+	const { data: products } = useProductsQuery();
+	const isPartnerOAuthTokenLoaded = useSelector( getIsPartnerOAuthTokenLoaded );
+	// eslint-disable-next-line no-unused-vars,@typescript-eslint/no-unused-vars
+	const { search, currentPage, filter, sort } = useContext( SitesOverviewContext );
+	const {
+		data: verifiedContacts,
+		refetch: refetchContacts,
+		isError: fetchContactFailed,
+	} = useFetchMonitorVerfiedContacts( isPartnerOAuthTokenLoaded );
+
+	const [ sitesViewState, setSitesViewState ] = useState< SitesViewState >( {
+		type: 'table',
+		perPage: 50,
+		page: currentPage,
+		sort: {
+			field: 'url',
+			direction: 'desc',
+		},
+		search: search,
+		filters:
+			filter?.issueTypes?.map( ( issueType ) => {
+				return {
+					field: 'status',
+					operator: 'in',
+					value: filtersMap.find( ( filterMap ) => filterMap.filterType === issueType )?.ref || 1,
+				};
+			} ) || [],
+		hiddenFields: [ 'status' ],
+		layout: {},
+		selectedSite: undefined,
+	} );
+	/*const { data, isError, isLoading, refetch } = useFetchDashboardSites(
+		isPartnerOAuthTokenLoaded,
+		search,
+		sitesViewState.page,
+		filter,
+		sort,
+		sitesViewState.perPage
+	);*/
+	//-- Mock data --// todo: fetch the sites from the API endpoint
+	const data: SitesDataResponse = {
+		sites: mockedSites,
+		total: 5,
+		per_page: 50,
+		total_favorites: 2,
```

Comment:
```suggestion
		perPage: 50,
		totaFavorites: 2,
```

I'm not 100% caught up here, but just notice that in my editor these two values are expected because that's what is defined in `SitesDataResponse`.

Reply:
It ended up being flagged by failing tests after fixing the tests below, so added: 10c2428db41379e2ee04330f8e265c8c622ffd45